plugins/ci/cmd/insights-ci/main.go

```diff
@@ -212,18 +212,23 @@ func getTrivyReport(images []trivymodels.Image, configurationObject models.Confi
 	if err != nil {
 		return trivyReport, err
 	}
-
+	refLookup := map[string]string{}
 	// Download missing images
 	for idx, currentImage := range images {
 		if currentImage.PullRef != "" {
 			continue
 		}
+		if ref, ok := refLookup[currentImage.Name]; ok {
```

Comment:
guessing `Name` has the sha, not just the tagname?

Reply:
It's the tagname (or whatever is specified in the manifest). But that's all we have in the CI/CD context. It's exactly the same thing we use for the skopeo copy below.

Reply:
👍